public/app/features/dashboard/components/GenAI/GenAIButton.test.tsx

```diff
@@ -1,10 +1,9 @@
-import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import { fireEvent, screen, waitFor } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
-import { Router } from 'react-router-dom';
 import { Observable } from 'rxjs';
+import { render } from 'test/test-utils';
```

Comment:
(minor, but you can also import `fireEvent, screen, waitFor` from these same utils if you like)